data_cleaning_job.ipynb

```diff
@@ -0,0 +1,2353 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # concatenate data for andaman and nicobar and delhi
</pre><p>I think this cell needs to be deleted</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
yup you are right